Frontend/CO.CDP.OrganisationApp.Tests/Pages/ShareInformation/ShareCodeConfirmationTests.cs

```diff
@@ -80,7 +80,7 @@ public async Task OnGetDownload_ShouldRedirectToPageNotFound_WhenFileIsNotFound(
 
         _dataSharingApiClientMock
             .Setup(x => x.GetSharedDataFileAsync(shareCode))
-            .ReturnsAsync((WebApiClient.FileResponse?)null);
+            .ReturnsAsync((WebApiClient.FileResponse?)null!);
```

Comment:
As discussed - this shouldn't return null - it should perhaps be throwing a 404 ApiException?